api/src/services/mail/index.ts

```diff
@@ -53,7 +54,16 @@ export class MailService {
 	}
 
 	async send<T>(options: EmailOptions): Promise<T> {
-		const { template, ...emailOptions } = options;
+		const payload = await emitter.emitFilter(`email.send`, options, {
+			database: getDatabase(),
+			schema: null,
+			accountability: null,
+		});
+
+		if (!payload) return null as T;
```

Comment:
We should probably adjust the method signature to return `Promise<T | null>`.

```suggestion
		if (!payload) return null;
```